includes/blocks.php

```diff
@@ -913,6 +940,22 @@ function ( $block ) {
 	}
 }
 
+
+/**
+ * Enqueues scripts and styles to load inside the block editor iframe.
+ * This allows us to do things like style contenteditable, and other inline editing elements.
+ *
+ * @since ACF 6.7
+ */
+function acf_enqueue_in_iframe_styles() {
+	if ( is_admin() ) {
+		$min      = defined( 'ACF_DEVELOPMENT_MODE' ) && ACF_DEVELOPMENT_MODE ? '' : '.min';
+		$css_path = acf_get_url( "assets/build/css/pro/acf-styles-in-iframe-for-blocks{$min}.css" );
+		wp_enqueue_style( 'acf-inline-editing-styles', $css_path, ACF_VERSION, true );
```

Comment:
Even if ported from upstream, this enqueuing doesn't seem to follow [the function signature](https://developer.wordpress.org/reference/functions/wp_enqueue_style/); it's passing `ACF_VERSION` as dependency and `true` as version. I'm not sure what `true` is meant to be here, but the call should probably be:

```suggestion
		wp_enqueue_style( 'acf-inline-editing-styles', $css_path, array(), ACF_VERSION );
```

Reply:
The first one will create this HTML

`<link rel="stylesheet" id="acf-inline-editing-styles-css" href="http://localhost:8888/wp-content/plugins/secure-custom-fields/assets/build/css/pro/acf-styles-in-iframe-for-blocks.min.css?ver=1" media="all">`

The one you provided will create this one:

`<link rel="stylesheet" id="acf-inline-editing-styles-css" href="http://localhost:8888/wp-content/plugins/secure-custom-fields/assets/build/css/pro/acf-styles-in-iframe-for-blocks.min.css?ver=6.7.1" media="1">`

Let's use this code:
`wp_enqueue_style( 'acf-inline-editing-styles', $css_path, array(), ACF_VERSION, 'all' );`

and return: 

`<link rel="stylesheet" id="acf-inline-editing-styles-css" href="http://localhost:8888/wp-content/plugins/secure-custom-fields/assets/build/css/pro/acf-styles-in-iframe-for-blocks.min.css?ver=6.7.1" media="all">`

That way we cover the media property and the versioning as it should.